packages/flame/lib/src/game/flame_game.dart

```diff
@@ -131,8 +131,8 @@ class FlameGame extends Component with Game {
 
   /// Whether a point is within the boundaries of the visible part of the game.
   @override
-  bool containsPoint(Vector2 p) {
-    return p.x > 0 && p.y > 0 && p.x < size.x && p.y < size.y;
+  bool containsLocalPoint(Vector2 p) {
+    return p.x >= 0 && p.y >= 0 && p.x < size.x && p.y < size.y;
```

Comment:
```suggestion
    return p.x >= 0 && p.y >= 0 && p.x <= size.x && p.y <= size.y;
```

Reply:
If the screen is 800x600, then it contains pixels `[0..799] x [0..599]`. So the right and bottom edges must be exclusive.

Reply:
Aah, ofc